README.md

```diff
@@ -56,3 +64,52 @@ on features for which two rows both have observed data.
 * `BiScaler`: Iterative estimation of row/column means and standard deviations to get doubly normalized
 matrix. Not guaranteed to converge but works well in practice. Taken from [Matrix Completion and Low-Rank SVD via Fast Alternating Least Squares](http://arxiv.org/abs/1410.2596).
 
+
+## Note about Multiple vs. Single Imputation
+(From `scikit-learn`'s documentation)
+
+In the statistics community, it is common practice to perform multiple imputations,
+generating, for example, ``m`` separate imputations for a single feature matrix.
+Each of these ``m`` imputations is then put through the subsequent analysis pipeline
+(e.g. feature engineering, clustering, regression, classification). The ``m`` final
+analysis results (e.g. held-out validation errors) allow the data scientist
+to obtain understanding of how analytic results may differ as a consequence
+of the inherent uncertainty caused by the missing values. The above practice
+is called multiple imputation.
+
+Our implementation of `IterativeImputer` was inspired by the R MICE
+package (Multivariate Imputation by Chained Equations) [1], but differs from
+it by returning a single imputation instead of multiple imputations.  However,
+IterativeImputer` can also be used for multiple imputations by applying
+it repeatedly to the same dataset with different random seeds when
+``sample_posterior=True``. A quick example:
```

Comment:
Huh! I didn't realize how much we're diverging from what MICE means. Maybe we should eventually add a MICE wrapper which runs multiple random seed IterativeImputer instances, passes their results to a user-supplied function, and averages the analysis results? 

Even if that's desirable, definitely for a future PR.

Reply:
Ya I was thinking the same, but I'm not sure if it's that simple. I think the imputation algo has to be sufficiently random in some sense I don't fully get.